dev/bots/service_worker_test.dart

```diff
@@ -42,13 +48,42 @@ Future<void> main() async {
   await runWebServiceWorkerTestWithCachingResources(headless: false, testType: ServiceWorkerTestType.withFlutterJs);
   await runWebServiceWorkerTestWithCachingResources(headless: false, testType: ServiceWorkerTestType.withFlutterJsShort);
   await runWebServiceWorkerTestWithCachingResources(headless: false, testType: ServiceWorkerTestType.withFlutterJsEntrypointLoadedEvent);
+  await runWebServiceWorkerTestWithGeneratedEntrypoint(headless: false);
   await runWebServiceWorkerTestWithBlockedServiceWorkers(headless: false);
+
   if (hasError) {
     print('One or more tests failed.');
     reportErrorsAndExit();
   }
 }
 
+// Regression test for https://github.com/flutter/flutter/issues/109093.
+//
+// Tests the entrypoint that's generated by `flutter create`.
+Future<void> runWebServiceWorkerTestWithGeneratedEntrypoint({
+  required bool headless,
+}) async {
+  await _generateEntrypoint();
+  await runWebServiceWorkerTestWithCachingResources(headless: headless, testType: ServiceWorkerTestType.generatedEntrypoint);
+}
+
+Future<void> _generateEntrypoint() async {
+  final Directory tempDirectory = Directory.systemTemp.createTempSync('flutter_web_generated_entrypoint.');
```

Comment:
Is the trailing `.` in the directory name intended or a typo?

Reply:
It's intended. I stole it from another test from `test.dart`. I'm now considering it a "convention" :)

Reply:
yeah, i think a hash gets appended, so the trailing `.` looks cleaner